frontend/include/chpl/uast/Attribute.h

```diff
@@ -33,27 +33,33 @@ class Attribute final: public AstNode {
   // the attribute name - deprecated or unstable or chpldoc.nodoc, for example
   UniqueString name_;
   int numActuals_; // number of child actuals
+  bool usedParens_; // whether the attribute was written with parens - only needed
+                    // while @unstable "msg" is going through deprecation from 1.30
   std::vector<UniqueString> actualNames_;
 
-  Attribute(UniqueString name, int numActuals, AstList actuals, std::vector<UniqueString> actualNames)
+  Attribute(UniqueString name, int numActuals, bool usedParens, AstList actuals,
+            std::vector<UniqueString> actualNames)
     : AstNode(asttags::Attribute, std::move(actuals)),
       name_(name),
       numActuals_(numActuals),
+      usedParens_(usedParens),
       actualNames_(std::move(actualNames)) {
   }
 
   Attribute(Deserializer& des)
     : AstNode(asttags::Attribute, des) {
       name_ = des.read<UniqueString>();
       numActuals_ = des.read<int>();
+      usedParens_ = des.read<bool>();
```

Comment:
Makes me wonder if we should encapsulate our fields in a macro to make this generally easier.